nautobot/dcim/models/device_components.py

```diff
@@ -980,6 +980,10 @@ def clean(self):
                     }
                 )
 
+    @property
+    def parent(self):
+        return self.device
+
```

Comment:
I'm not seeing this in `DeviceBay` on `next` or `develop`

Reply:
Was an incorrect removal in `next` - `develop` added a new test that exercises code that requires every device-component (as well as `VMInterface`) to have a `.parent` property. We didn't have this test in `next` before now so it was broken but we didn't know it.